backend/models/vueformGenerator.ts

```diff
@@ -52,15 +52,25 @@ function mapSchemaTypeToVueformElement(
     vueformElement['rules'].push('max:' + schemaType.max)
   }
 
-  if (schemaType.description !== undefined) {
-    vueformElement['description'] = translate('description.' + labelStr, language)
+  if (schemaType.description) {
+    vueformElement['description'] = translate(schemaType.description, language)
+  } else if (labelStr) {
+    let description = translate('description.' + labelStr, language) as { de: string; en: string }
+    if (description.de) {
+      console.log(description.de)
+      console.log('description.' + labelStr)
+      if (!(description.de === 'description.' + labelStr)) {
+        vueformElement['description'] = translate('description.' + labelStr, language)
+      }
```

Comment:
Die if Bedingung habe ich hinzugefügt, um zu verhindern, dass für alle Schematypen eine Description angezeigt wird, die nur aus dem Translationkey besteht, wenn in den Locales nichts eingetragen ist.

Reply:
Warum machst du es so kompliziert?

Mach es doch so wie es beim `schemaType.label` ist.

Du schreibst in dein Schema bei description einfach den translation key rein, und den nimmst du dann immer, den Fallback für wenn es keinen gibt wird nicht gebraucht.